pimpmycause/profiles/views.py

```diff
@@ -42,12 +44,26 @@ class TermsAndConditions(TemplateView):
     template_name = 'core/terms_and_conditions.html'
 
 
+
 def logout_view(request):
     logout(request)
 
     return HttpResponseRedirect("/")
 
 
+def homepage(request):
+    """
+    The home page view.
+    """
+    limit = 3
+    featured_marketer_list = PimpUser.objects.filter(featured=True).filter(usertype=0).order_by('-date_joined')[:limit]
+    featured_cause_list = PimpUser.objects.filter(featured=True).filter(usertype=1).order_by('-date_joined')[:limit]
```

Comment:
`usertype=0` is not very readable, I'll suggest using `usertype = PimpUser.MARKETER`, is more readable and more DRY (in case you want to convert from a numeric based system to a string based system).
Since the constants are already defined inside the model you don't even need to import them.